README.md

```diff
@@ -143,28 +143,24 @@ flyway -user=$USER -password="" -locations="filesystem:database/sql/" -url=jdbc:
 
 ### Local Test Network
 
-In order to run the `e2e` tests you have to have a testnet running locally.
-Due to the RPC calls the services issue `Ganache` is incompatible, so we will use `hardhat`.
-
-1. Install [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
-2. Install hardhat with `npm install --save-dev hardhat`
-3. Create `hardhat.config.js` in the directory you installed `hardhat` in with following content:
-   ```javascript
-   module.exports = {
-       networks: {
-           hardhat: {
-               initialBaseFeePerGas: 0,
-               initialDate: "2000-01-01T00:00:00.000+00:00",
-               accounts: {
-                   accountsBalance: "1000000000000000000000000"
-                  },
-                  gas: 1e7,
-                  gasPrice: 1
-           }
-       }
-   };
-   ```
-4. Run local testnet with `npx hardhat node`
+In order to run the `e2e` tests you have to have an EVM compatible testnet running locally.
+We make use of [anvil](https://github.com/foundry-rs/foundry) from the Foundry project to spin up a local testnet.
```

Comment:
@devanoneth I remember that your tests made use of our test setup logic.
Could you please check out this branch and see if this breaks things on your end? Would be interesting to see if replacing `hardhat` has more consequences than we are able to see with our tests.